kore/test/Test/Kore/Step/Simplification/Equals.hs

```diff
@@ -184,6 +194,45 @@ test_equalsSimplification_Or_Pattern =
                         , substitution = mempty
                         }
                     ]
+            expectSimplified =
```

Comment:
This should be a completely separate test in `Test.Kore.Step.Simplification.Pattern`. There we should check that the `Pattern` simplifier causes the `\implies` to be simplified, but we don't need to check the original `\equals` simplification.

Reply:
I moved all the tests like this to `Test.Kore.Step.Simplification.Pattern`.